bct/utils/visualization.py

```diff
@@ -307,39 +309,45 @@ def backbone_wu(CIJ, avgdeg):
     CIJtree = np.zeros((n, n))
 
     # find strongest edge (if multiple edges are tied, use only first one)
-    i, j = np.where(np.max(CIJ) == CIJ)
-    im = [i[0], i[1]]  # what?  why take two values?  doesnt that mess up multiples?
-    jm = [j[0], j[1]]
+    im, jm = np.unravel_index(np.argmax(CIJ, axis=None), CIJ.shape)
```

Comment:
This was indeed a bug as pointed out in the comment. You can see the fix in https://github.com/aestrivex/bctpy/pull/112/commits/40b4f2c, after which I also refactored the code which will produce only the strongest edge.